src/modal.js

```diff
@@ -77,6 +77,18 @@ export function VidModal({ isOpen, onClose, vidSource, setBackTime }) {
     }
   };
 
+  function exportTimeframe(){
```

Comment:
once we call on create how are we clearing the state of the modal? might make sense to clear it after the submit and make a call to onCancel to close the modal.  if you open another modal now won't it still have the state from when you submitted?